lib/notesclub_web/controllers/status_controller.ex

```diff
@@ -1,5 +1,17 @@
 defmodule NotesclubWeb.StatusController do
   use NotesclubWeb, :controller
 
+  import Notesclub.Notebooks, only: [get_latest_notebook: 0]
+
   def ok(conn, _params), do: text(conn, "OK")
```

Comment:
Should we remove this endpoint? I don't see the need for it anymore.

Reply:
Thanks for this PR!

Sure, let's remove it. Let me know if you wish me to do it.

Reply:
I will do it, no worries 🙂

Reply:
Thanks!

Reply:
Done, please have another look